component/vshn_postgres.jsonnet

```diff
@@ -841,6 +841,59 @@ local maintenanceJob = {
   ] + convertToCron(),
 };
 
+local prometheusRule = {
+  base: comp.KubeObject('monitoring.coreos.com/v1', 'PrometheusRule') + {
+    spec+: {
+      forProvider+: {
+        manifest+: {
+          metadata: {
+            name: 'postgresql-storage-rules',
+          },
+          spec: {
+            groups: [
+              {
+                name: 'postgresql-storage',
+                rules: [
+                  {
+                    alert: 'PostgreSQLPersistentVolumeFillingUp',
+                    annotations: {
+                      description: 'The PersistentVolume claimed by {{ $labels.persistentvolumeclaim\n              }} in Namespace {{ $labels.namespace }} is only {{ $value |\n              humanizePercentage }} free.',
```

Comment:
Does this actually work with a newline in the template?

Reply:
No idea if opsgenie will respect them, but in the applied yaml file on the cluster they are there.